### PR TITLE
Add product support for gumgum

### PR DESCRIPTION
**Pull request by gg-natalia**
IN this changes we will start to pass the product param from Prebid Server to GG Ad Server under the impressions.ext param.
For any questions/comments pls feel free to reach out
Thanks
Natalia Castiglioni

### CODE REVIEW

File: src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java [hunk 1]
```diff
@@ -116,6 +118,13 @@ private ExtImpGumgum parseImpExt(Imp imp) {
 
     private Imp modifyImp(Imp imp, ExtImpGumgum extImp) {
         final Imp.ImpBuilder impBuilder = imp.toBuilder();
+        //Product parameter
```
Comment (SerhiiNahornyi) [inline src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java:121]: Comment is redundant, please remove

**Review by SerhiiNahornyi — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java (hunk 1 above)
Comment (gg-natalia) [reply src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java]: Changed, ty

**Review by SerhiiNahornyi — APPROVED** [review]
LGTM

**Review by SerhiiNahornyi — COMMENTED** [review]
[INFO] Starting audit...
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java:127: Line has trailing spaces. [RegexpSingleline]
Audit done.


You can check build locally by running `mvn clean package -f extra` even `mvn clean package` will be enough

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**